src/core_atmosphere/CMakeLists.txt

```diff
@@ -2,165 +2,354 @@
-        mpas_atmphys_vars.F
+    ccpp_kind_types.F
+    mpas_atmphys_camrad_init.F
+    mpas_atmphys_constants.F
+    mpas_atmphys_control.F
+    mpas_atmphys_date_time.F
+    mpas_atmphys_driver_cloudiness.F
+    mpas_atmphys_driver_microphysics.F
+    mpas_atmphys_driver_oml.F
+    mpas_atmphys_finalize.F
+    mpas_atmphys_functions.F
+    mpas_atmphys_init_microphysics.F
+    mpas_atmphys_interface.F
+    mpas_atmphys_landuse.F
+    mpas_atmphys_lsm_noahinit.F
+    mpas_atmphys_manager.F
+    mpas_atmphys_o3climatology.F
+    mpas_atmphys_rrtmg_lwinit.F
+    mpas_atmphys_rrtmg_swinit.F
+    mpas_atmphys_update.F
+    mpas_atmphys_update_surface.F
+    mpas_atmphys_utilities.F
+    mpas_atmphys_driver.F
+    mpas_atmphys_driver_convection.F
+    mpas_atmphys_driver_gwdo.F
+    mpas_atmphys_driver_lsm.F
+    mpas_atmphys_driver_pbl.F
+    mpas_atmphys_driver_radiation_lw.F
+    mpas_atmphys_driver_radiation_sw.F
+    mpas_atmphys_driver_seaice.F
+    mpas_atmphys_driver_sfclayer.F
+    mpas_atmphys_init.F
+    mpas_atmphys_lsm_shared.F
+    mpas_atmphys_packages.F
+    mpas_atmphys_todynamics.F
+    mpas_atmphys_vars.F
+    mpas_atmphys_driver_lsm_noahmp.F
+    mpas_atmphys_lsm_noahmpfinalize.F
+    mpas_atmphys_lsm_noahmpinit.F
 )
 list(TRANSFORM ATMOSPHERE_CORE_PHYSICS_SOURCES PREPEND physics/)
 
 ## Unused
 # physics/physics_wrf/
 set(ATMOSPHERE_CORE_PHYSICS_WRF_SOURCES
-        libmassv.F
-        module_bep_bem_helper.F
-        module_bl_gwdo.F
-        module_bl_ysu.F
-        module_cam_error_function.F
-        module_cam_shr_kind_mod.F
-        module_cam_support.F
-        module_cu_gf.mpas.F
-        module_mp_kessler.F
-        module_mp_radar.F
-        module_mp_thompson.F
-        module_mp_thompson_cldfra3.F
-        module_mp_wsm6.F
-        module_ra_cam_support.F
-        module_ra_rrtmg_lw.F
-        module_ra_rrtmg_sw.F
-        module_ra_rrtmg_vinterp.F
-        module_sf_bem.F
-        module_sf_bep.F
-        module_sf_bep_bem.F
-        module_sf_noah_seaice.F
-        module_sf_noah_seaice_drv.F
-        module_sf_noahdrv.F
-        module_sf_noahlsm.F
-        module_sf_noahlsm_glacial_only.F
-        module_sf_oml.F
-        module_sf_sfcdiags.F
-        module_sf_sfclay.F
-        module_sf_sfclayrev.F
-        module_sf_urban.F
-        bl_mynn_post.F
-        bl_mynn_pre.F
-        module_bl_mynn.F
-        module_cu_kfeta.F
-        module_cu_ntiedtke.F
-        module_cu_tiedtke.F
-        module_ra_cam.F
-        module_sf_mynn.F
-        sf_mynn_pre.F
+    libmassv.F
+    cu_ntiedtke_post.F
+    cu_ntiedtke_pre.F
+    module_bep_bem_helper.F
+    module_bl_gwdo.F
+    module_bl_ysu.F
+    module_cam_error_function.F
+    module_cam_shr_kind_mod.F
+    module_cam_support.F
+    module_cu_gf.mpas.F
+    module_mp_kessler.F
+    module_mp_radar.F
+    module_mp_thompson.F
+    module_mp_thompson_cldfra3.F
+    module_mp_thompson_aerosols.F
+    module_mp_wsm6.F
+    module_ra_rrtmg_sw_aerosols
```

Comment:
I think this filename is missing a suffix. (Also, in my opinion, this would have been easier to spot if so many whitespace changes were not introduced.)

Reply:
Good catch! Also you can hide white space changes. I always do this and find it to be really helpful.